pkgs/top-level/all-packages.nix

```diff
@@ -18609,6 +18609,8 @@ in
 
     gcadapter-oc-kmod = callPackage ../os-specific/linux/gcadapter-oc-kmod { };
 
+    hid-nintendo = if builtins.elem kernel.version ["libre" "4.4" "4.9"] then null else callPackage ../os-specific/linux/hid-nintendo { };
```

Comment:
Please do not overwrite it with null which causes eval error. Broken should cover it.